openpype/hosts/houdini/api/lib.py

```diff
@@ -649,3 +649,60 @@ def get_color_management_preferences():
         "display": hou.Color.ocio_defaultDisplay(),
         "view": hou.Color.ocio_defaultView()
     }
+
+
+def get_current_asset_doc():
```

Comment:
Please do not duplicate already existing functions. There is `get_current_project_asset` in `openpype.pipeline.context_tools`.